src/main/java/io/ncbpfluffybear/supserv/SupServItemSetup.java

```diff
@@ -76,6 +78,14 @@ public static void setup(@Nonnull SupServPlugin plugin) {
             }, new ItemStack(Material.DRAGON_EGG)
         ).register(plugin);
 
+        new FarmersHoe(SupServItems.SUPSERV_CATEGORY, SupServItems.FARMERS_HOE, RecipeType.ANCIENT_ALTAR,
+                new ItemStack[] {
+                        SlimefunItems.EARTH_RUNE, SlimefunItems.WATER_RUNE, SlimefunItems.EARTH_RUNE,
+                        new ItemStack(Material.BONE_BLOCK), new ItemStack(Material.IRON_HOE), new ItemStack(Material.BONE_BLOCK),
+                        SlimefunItems.EARTH_RUNE, SlimefunItems.WATER_RUNE, SlimefunItems.EARTH_RUNE,
+                }, new SlimefunItemStack(SupServItems.FARMERS_HOE, 1)
```

Comment:
You don't need to specify the recipe output unless the stack size is greater than 1 or the item is not the same as specified in the second parameter.